src/Compilers/CSharp/Portable/Compiler/Instrumentation.cs

```diff
@@ -109,6 +115,30 @@ private static bool IsTestMethod(MethodSymbol method)
             // PROTOTYPE (https://github.com/dotnet/roslyn/issues/9811): Make this real. 
             return method.Name.StartsWith("Test");
         }
+
+        internal sealed partial class SynthesizedPrivateImplementationStaticConstructor : SynthesizedGlobalMethodSymbol
+        {
+            internal SynthesizedPrivateImplementationStaticConstructor(SourceModuleSymbol containingModule, PrivateImplementationDetails privateImplementationType, SyntheticBoundNodeFactory factory)
```

Comment:
Rather than passing factory with only purpose to find symbol for void type, consider passing the symbol instead.